input/input_test.go

```diff
@@ -277,3 +278,51 @@ func BenchmarkProcessMetricDataSameMetric(b *testing.B) {
 		in.ProcessMetricData(datas[i], 1)
 	}
 }
+
+func BenchmarkProcessMetricPoint(b *testing.B) {
```

Comment:
I don't like how inconsistent this function is compared to the existing BenchmarkProcessMetricData functions
e.g. if you're going to introduce aggregators to slow down the adds, that should be added to the function name to make it clear.

Reply:
actually, should it be the inputs package's concern to benchmark the performance of mdata's addAggregators functionality?

mdata already has `BenchmarkAggMetricAdd` maybe we should make another similar one that has aggregators enabled